app/composables/dashboard/useDcBaseChart.ts

```
@@ -0,0 +1,168 @@
+      ret = [filter]; // 単一選択
+    } else {
+      filters.push(filter); // 既存のフィルタに追加
+      ret = filters; // 追加選択
+    }
+    isMouseLongClick.value = false; // フラグをリセット
+    return ret;
+  };
+
+  // マウスイベントの設定
+  const setupMouseEvents = () => {
+    const container = document.getElementById(chartId);
```

ESLint

[error] Extra semicolon.

(@stylistic/semi)